services/brig/src/Brig/Options.hs

```diff
@@ -519,6 +524,26 @@ data CustomerExtensions = CustomerExtensions
 newtype DomainsBlockedForRegistration = DomainsBlockedForRegistration [Domain]
   deriving newtype (Show, FromJSON, Generic)
 
+data SFTOptions = SFTOptions
+  { sftBaseDomain :: !DNS.Domain,
+    sftSRVServiceName :: !(Maybe ByteString),
```

Comment:
Isn't the service name always `_sft` here? Do we need to make this configurable? The defaults/ fallbacks on not specifying would be nice to document, or move closer to the data type, as it's currently hard to see what these values will be when not set.

We have some `defSomething` values at the bottom of this file, can we add the `_sft` and `10 seconds` here as well?

Reply:
Done in fde677111beaecd2d77b63a86e024cbe62ca79a8